package.json

```diff
@@ -13,8 +13,8 @@
     "Utilities/"
   ],
   "dependencies": {
-    "@patternfly/react-core": "^1.51.4",
-    "@patternfly/react-icons": "^2.8.0",
+    "@patternfly/react-core": "^1.52.0",
```

Comment:
Is there a reason to bump `@patternfly/react-core` and `@patternfly/react-icons` dependencies?

Reply:
Yes, we are using features like `<Tooltip>` from new version.

Reply:
Can you identify which packages are required to make this running? `@patternfly/react-core`, `@patternfly/react-icons` and `react-dom`? Anything else? I will create new PR that updates these packages so we have slightly better consistency.

Reply:
OK, please make a new PR with these changes:
```
"@patternfly/react-core": "^1.52.0",
"@patternfly/react-icons": "^2.10.0",
"react-dom": "^16.7.0",

```

Reply:
Sure thing! New PR has been opened https://github.com/RedHatInsights/insights-frontend-components/pull/317